### PR TITLE
Docs-1339: 3.16.1

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s): Calico Enterprise GA

**PM** 
[ x ] RN new features and improvements 

**Dev**
[ x ] release.json updates to Tigera operator and component versions 
[ x ] Review RN Known Issues (add/remove) 
   
**Writer**
 [  ] Verify manifests after GA 
 [ x ] Waiting for docs: ALP, WAF, SELinux
 [ x ] Update date on RN
 [ x ] Verify manifest URLs in install docs after GA is pushed before merging PR
 [ x ] Edit and grammar check RN 
 [ x ] Verify version drop-down 
 [ x ] release.json - add new array for version
 [ x ] variable.js -  Update releaseTitle, filesURL, chart_version_name, manifestsURL
 [ x ] docusaurus.js - Change text from “Preview” to “Latest"
 [ x ] Run script to update latest: ./switch-latest.sh calico enterprise 3.15 3.16

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.16/_includes/release-notes/_v3.16.1-release-notes.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+{{prodname}} has updated its components to apply a more restrictive `SecurityContext` where applicable.
+This includes the following:
+* Non-root context whenever possible and added drop ALL `Capabilities`
+* Root context and privilege escalation are used only when necessary
+* `SeccompProfile` is set to `RuntimeDefault`
+
+## Known issues
+
+* L7 log collection fails to deploy on CIS hardened clusters. As a result, some cards in the manager UI dashboard will not display any metrics.
```
Comment (hjiawei) [inline calico-enterprise_versioned_docs/version-3.16/_includes/release-notes/_v3.16.1-release-notes.mdx]: This issue is resolved and can be removed.

File: calico-enterprise_versioned_docs/version-3.16/releases.json [hunk 2]
```diff
@@ -1,5 +1,257 @@
...
+      "fluentd-windows": {
+        "image": "tigera/fluentd-windows",
+        "version": "v3.16.1"
+      },
+      "es-proxy": {
+        "image": "tigera/es-proxy",
+        "version": "v3.16.1"
+      },
+      "eck-kibana": {
+        "version": "7.17.7"
```
Comment (hjiawei) [inline calico-enterprise_versioned_docs/version-3.16/releases.json]: Kibana is upgraded to 7.17.9.

File: calico-enterprise_versioned_docs/version-3.16/releases.json [hunk 3]
```diff
@@ -1,5 +1,257 @@
...
+      },
+      "kibana": {
+        "image": "tigera/kibana",
+        "version": "v3.16.1"
+      },
+      "eck-elasticsearch": {
+        "version": "7.17.7"
```
Comment (hjiawei) [inline calico-enterprise_versioned_docs/version-3.16/releases.json]: Elasticsearch is upgraded to 7.17.9.

File: calico-enterprise_versioned_docs/version-3.16/releases.json [hunk 4]
```diff
@@ -1,5 +1,257 @@
...
+      "es-gateway": {
+        "image": "tigera/es-gateway",
+        "version": "v3.16.1"
+      },
+      "deep-packet-inspection": {
+        "image": "tigera/deep-packet-inspection",
+        "version": "v3.16.1"
+      },
+      "eck-elasticsearch-operator": {
+        "version": "2.5.0"
```
Comment (hjiawei) [inline calico-enterprise_versioned_docs/version-3.16/releases.json:209]: eck operator is upgraded to 2.6.1

File: calico-enterprise_versioned_docs/version-3.16/_includes/release-notes/_v3.16.1-release-notes.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.16/_includes/release-notes/_v3.16.1-release-notes.mdx]: Deleted. Thanks!

File: calico-enterprise_versioned_docs/version-3.16/releases.json (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.16/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.16/releases.json (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.16/releases.json:209]: Fixed.

File: docusaurus.config.js [hunk 5]
```diff
@@ -374,13 +374,13 @@ const config = {
             banner: 'unreleased',
           },
           3.16: {
-            label: '3.16 (early preview)',
-            path: '3.16',
-            banner: 'unreleased',
+            label: '3.16 (Latest)',
```
Comment (ctauchen) [inline docusaurus.config.js]:
```suggestion
            label: '3.16 (latest)',
```

Comment (ctauchen) [reply docusaurus.config.js]: I can't add a comment above in the right place:

LINE 369:
`lastVersion: '3.15',` ==> `lastVersion: '3.16',`

**Review by ctauchen — DISMISSED** [review]
Two smallish things, plus a squash! Otherwise LGTM

File: calico-enterprise_versioned_docs/version-3.16/releases.json (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.16/releases.json]: Fixed.

File: docusaurus.config.js (hunk 5 above)
Comment (ctauchen) [reply docusaurus.config.js]: > I can't add a comment above in the right place:
> 
> LINE 369: `lastVersion: '3.15',` ==> `lastVersion: '3.16',`

This still needs fixing.

Comment (bmckercher123) [reply docusaurus.config.js]: Fixed.

Comment (ctauchen) [reply docusaurus.config.js]: The fix hasn't made it in somehow. Needs to look like this: 
image

Comment (ctauchen) [reply docusaurus.config.js]: Took me a while to realize this was what was causing a strange problem with your preview build. The entire 3.16 doc set was taking variables.js from the 3.15 docs: 
image
Note this is 3.16 docs, but with 3.15 information.

File: calico-enterprise/releases.json
Comment (ctauchen) [inline calico-enterprise/releases.json:1]: Why are you making changes here? Unless there's a compelling reason, I think all changes in this doc should be reverted.

File: docusaurus.config.js (hunk 5 above)
Comment (bmckercher123) [reply docusaurus.config.js]: Odd thing is, I did fix this early yesterday and then made updates to RN. But there is another odd thing. Netlify view shows correct version of RN, and so does local, but git page is not updated. I've rebased to see if things will sync up.

File: calico-enterprise/releases.json
Comment (bmckercher123) [reply calico-enterprise/releases.json:1]: Are you saying I should not change "master" to "3.16.1"? The updates are to reflect the changes given to me by Rene. ??

Comment (ctauchen) [reply calico-enterprise/releases.json:1]: This `releases.json` is for version Next, not 3.16. Without compelling reason to mirror a specific version, we should always leave these as 'master'.

File: docusaurus.config.js [hunk 6]
```diff
@@ -375,7 +375,7 @@ const config = {
           },
           3.16: {
             label: '3.16 (latest)',
-            path: 'latest',
+            path: '3.16',
```
Comment (doucol) [inline docusaurus.config.js]: I think this should be 'latest'.

**Review by doucol — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 5 above)
Comment (bmckercher123) [reply docusaurus.config.js]: Fixed.

File: calico-enterprise/releases.json
Comment (bmckercher123) [reply calico-enterprise/releases.json:1]: Reverted

File: docusaurus.config.js (hunk 6 above)
Comment (bmckercher123) [reply docusaurus.config.js]: Fixed.

File: calico-enterprise_versioned_docs/version-3.16/variables.js [hunk 7]
```diff
@@ -1,12 +1,12 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.16.0',
+  releaseTitle: 'v3.16.1',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.16',
   baseUrl: '/calico-enterprise/3.16',
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.16/variables.js]: @bmckercher123 The missing bit is this variable, which needs to be changed both here and in the 3.15 variables.js. Here, it needs to be changed to '/latest'. In 3.15, it needs to be changed from '/latest' to '/3.15'

(All of this happens automatically with the script, but you can also change it manually.

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.16/variables.js]: I ran the script and the entries seem fine. However, the 3.15 version says "3.15.2" so if the script updated this, I assume it is correct.

**Comment by bmckercher123** [comment]
This PR is clean version of this one: https://github.com/tigera/docs/pull/534.

**Pull request closed without merging**